fdi/language/tr/FDSETUP.DEF

```diff
@@ -166,12 +165,12 @@ KBL=" Daha az klavye se
 KBA_FRAME=/w58 /h13 /c
 
 # FDASK200 - Backup old OS
-BACKUP_FRAME=/w70 /h16 /c
-BACKUP_OPTS=/w45 /h5 /c
-BACKUPADV_FRAME=/w64 /h11 /c
+BACKUP_FRAME=/w70 /h10 /c
+BACKUP_OPTS=/w45/h5 /c
+BACKUPADV_FRAME=/w70 /h11 /c
 BACKUPADV_OPTS=/w45 /h5 /c
-BACKUP=/f %1 %2 /f %3 " s�r�c�s�nde kurulu bir i�letim sistemi bulundu."
-BACKUP?="Kurulumdan �nce mevcut dosyalar� yedeklemek ister misiniz?"
+BACKUP=/f %1 %2 /f %3 /s- s�r�c�s�nde kurulu bir i�letim sistemi bulundu.

```

Comment:
You do not want to use the `/s-` switch here. That switch turns off spacing between parameter strings and would cause all the words to run together as one. I'm not sure why there was a space in the original quoted string. But, it should be fine without it. So either of these should be fine...

`BACKUP=/f %1 %2 /f %3 s�r�c�s�nde kurulu bir i�letim sistemi bulundu.
`
or 
`BACKUP=/f %1 %2 /f %3 "s�r�c�s�nde kurulu bir i�letim sistemi bulundu."`